scripts/buildDocs.sh

```diff
@@ -1,12 +1,37 @@
 #!/bin/bash
 set -e
 
+# Store docker image name
+DOCKER_IMAGE=ui5-tooling/mkdocs-material
+GIT_COMMITTER_NAME: "OpenUI5 Bot"
+GIT_COMMITTER_EMAIL: "openui5@sap.com"
```

Comment:
this is valid bash? Never saw this syntax before

Reply:
No, copied it from the GH workflow and pushed a bit too early :)